index.html

```diff
@@ -35,10 +35,215 @@ <h1 class="developers-name">Charlie Giorgania</h1>
+          <div class="works-skills">
+            <ul class="skills-list">
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="works-bttn">
+              <a href="#" class="works-btn-styles">See this project</a>
+              <img
+                src="/resources/images/arrow.png"
+                alt="arrow image"
+                class="arrow-image"
+              />
+            </div>
+          </div>
+        </div>
+        <div class="project-intro">
+          <img
+            src="/resources/images/graphicscomputerleft.png"
+            alt="Graphics in Computer"
+          />
+        </div>
+        <div class="works-info">
+          <h3>Project name goes here</h3>
+          <div class="works-skills">
+            <ul class="skills-list">
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="works-bttn">
+              <a href="#" class="works-btn-styles">See this project</a>
+              <img
+                src="/resources/images/arrow.png"
+                alt="arrow image"
+                class="arrow-image"
+              />
+            </div>
+          </div>
+        </div>
+        <div class="project-intro">
+          <img
+            src="/resources/images/graphicscomputerright.png"
+            alt="Graphics in Computer"
+          />
+        </div>
+        <div class="works-info">
+          <h3 class="sections-headers">Project name goes here</h3>
+          <div class="works-skills">
+            <ul class="skills-list">
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="works-bttn">
+              <a href="#" class="works-btn-styles">See this project</a>
+              <img
+                src="/resources/images/arrow.png"
+                alt="arrow image"
+                class="arrow-image"
+              />
+            </div>
+          </div>
+        </div>
+        <div class="project-intro">
+          <img src="/resources/images/pinkcomputer.png" alt="Pink Computer" />
+        </div>
+        <div class="works-info">
+          <h3>Project name goes here</h3>
+          <div class="works-skills">
+            <ul class="skills-list">
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="works-bttn">
+              <a href="#" class="works-btn-styles">See this project</a>
+              <img
+                src="/resources/images/arrow.png"
+                alt="arrow image"
+                class="arrow-image"
+              />
+            </div>
+          </div>
+        </div>
+        <div>
+          <div class="about-content">
+            <h2 class="main-topic topic-h2">About</h2>
```

Comment:
- [ ] Please kindly make sure that that about section uses an enclosing `section` tag.